data/hai/hai reveal 1 intro.txt

```diff
@@ -0,0 +1,947 @@
+					accept
+	npc
+		government "Hai"
+		personality escort heroic
+		fleet "Anti-Pirate Hai (No Wormhole Travel)" 5
+	npc
+		government "Pirate"
+		personality staying disables plunders harvests
+		system "Waypoint"
+		fleet "Marauder fleet IX" 4
+		fleet "Hired Guns" 1
+	npc kill
+		government "Pirate"
+		personality staying marked target
+		system "Waypoint"
+		ship "Marauder Leviathan (Weapons)" "Black Lich"
+	to complete
+		or
+			has "00 Hai Leaks Response 4B: declined"
+			has "Hai Leaks Response 4A: failed"
+	on complete
+		payment 1000000
+		event "waypoint pirates vanquished"
+		log `Defeated a nasty marauder whose ship was the "Black Lich". He had managed to kidnap a number of Hai straight off the planet with a combination of trickery and speed. You were rewarded well for this particular piece of aid.`
+		conversation
+			`When you return the Hai Captain thanks you profusely, "Please, the Allhome government has authorised a reward of <payment> for routing the problem."`
+			branch "saved geocoris"
+				has "00 Hai Leaks Response 4B: declined"
+			`	He sighs deeply, "It is a shame though that the kidnapped Hai could not be saved, but the other Hai captains tell me it was a mess out there."`
+			`	He places one of his furry hands comfortingly on your shoulder, "I'm sure you did all you could."`
+				goto end
+			label "saved geocoris"
+			`	"Your rescue of our kidnapped people is deeply appreciated also. Lucky for you, the merchant who owned the Geocoris has already filed the insurance claim, so you get to keep the ship as a trophy."`
+			`	"I do need to do a full security scan of it before I let it go though, just to complete all the incident paperwork, so if you have any business on world here then you should take your time."`
+			label end
+			`	"In any event, let us hope a more permanent solution can be devised soon. In the meantime, I'm sure they'll be waiting for you back on Hai-home before too long."`
+			`	With a final nod of thanks he departs to attend to business, leaving you free to wander the spaceport until they release your new ship.`
+	
+	
+event "waypoint pirates vanquished"
+	#Update pirate and anti-pirate spawns in Waypoint and Heia Due
+	system "Waypoint"
+		government "Hai"
+		add fleet "Anti-Pirate Hai (Wormhole Travel)" 1000
+		remove fleet "Marauder fleet IX"
+		remove fleet "Anti-Pirate Hai (No Wormhole Travel)"
+		remove fleet "Large Northern Pirates"
+		remove fleet "Large Core Pirates"
+	system "Heia Due"
+		remove fleet "Marauder fleet IX"
+		remove fleet "Large Northern Pirates"
+		remove fleet "Large Core Pirates"
+
+# It is impossible to implement a "capture and return npc" mission. 
+# Once the ship is captured, the NPC is gone and the mission is failed.
+# The NPC has an outfit used to determine if you managed to get it back safely.
+mission "Hai Leaks Response 4A"
+	name "Recover the Geocoris"
+	description "Recover the stolen Geocoris supporting the pirates in Waypoint, and return to Allhome."
```

Comment:
```suggestion
	description "Recover the stolen Geocoris supporting the pirates in Waypoint, and return to Allhome. (You will need to swap the Geocoris to your flagship, and depart and land again in order to progress the missions properly.)"
```
I'd recommend putting this note here as well just to be safe in case the player accidentally skips over some dialogue.